index.html

```diff
@@ -0,0 +1,256 @@
+            <input 
+            type="text" 
+            id="Name" 
+            classe="form-control"
+            placeholder="Enter Your Name" 
+            required
+            />
+
+        </div>
+
+        <div class="form-group">
+            <label for="Email" id="Email-label">Email</label>
+            <input 
+              type="text" 
+              id="Email" 
+              classe="form-control" 
+              placeholder="Enter Your Email" 
+              required
+            />
+
+        </div>
+
+        <div class="form-group">
+            <label for="number" id="number-label">Age (optional)</label>
+            <input 
+            type="number" 
+            name="age"
+            id="number" 
+            min="10"
+            max="99"
+            classe="form-control" 
+            placeholder="Age" 
+            required
+            />
+
+        </div>
+
+        <div class="form-group">
+            <label for="dropdown">
+                Which option best describes your current role?
+            </label>
+            <select name="dropdown" id="dropdown" classe="form-control"required>
+            <option disabled>select current role</option>
+            <option value="Student">Student</option>
+            <option value="Full Time Job">Full Time Job</option>
+            <option value="Full Time Learner">Full Time Learner</option>
+            <option value="Prefer not to say">Prefer not to say</option>
+            <option value="Other">Other</option>
+            </select> 
+        </div>
+
+        <div class="form-group">
+            <p>Would you recommend freeCodeCamp to a friend?</p>
+            <p>
+                <label>
+                    <input type="radio" name="answer" id="definitely" value="definitely" checked/>
+                    definitely
+                </label>
+            </p>
+        
+            <p>
+                <label>
+                    <input type="radio" name="answer" id="maybe" value="maybe" checked/>
+                    maybe
+                </label>
+            </p>
+        
+            <p>
+                <label>
+                    <input type="radio" name="answer" id="not sure" value="not sure" checked/>
+                    not sure
+                </label>
+            </p>
+        </div>
+        
+        <div class="form-group">
+                <label for="favorite">
+                    What is your favorite feature of freeCodeCamp?
+                </label>
+                <select name="favorite" id="favorite" class="form control" required>
+                    <option disabled>Select an option</option>
+                    <option value="Challenges">Challenges</option>
+                    <option value="Projects">Projects</option>
+                    <option value="Community">Community</option>
+                    <option value="Open Source">Open Source</option>
+                </select>
```

Comment:
Good job so far!
Please kindly modify some issues that you still need to modify in the selection option from the dropdown list. one of the dropdown items lists is disabled. we shall change to  placeholder  "select current role" and modify the dropdown options  list item by adding a placeholder